apps/ejabberd/src/mongoose_subhosts.erl

```diff
@@ -0,0 +1,65 @@
+%%%----------------------------------------------------------------------
+%%% File    : mongoose_subhosts.erl
+%%% Author  : Piotr Nosek <piotr.nosek@erlang-solutions.com>
+%%% Purpose : SubHost->Host register
+%%% Created : 9 Dec 2016
+%%%----------------------------------------------------------------------
+
+-module(mongoose_subhosts).
+
+-include("ejabberd.hrl").
+
+-export([
+         init/0,
+         stop/0,
+
+         'register'/2,
```

Comment:
Didn't check. But it's annoying to see it highlighted in editor as BIF anyway. :)